src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -48,6 +48,17 @@ private IGeoMasterClient InitClient()
             return geoMasterClient;
         }
 
+        public string RemovePIIFromSettings(string content)
```

Comment:
I'd make this static and add to a static utility class

Reply:
So this function is a specific scenario to the GetAppSettings of this class . Not sure whether it will be useful to be added to a static utility class.